modules/accredited_representative_portal/spec/requests/accredited_representative_portal/v0/power_of_attorney_requests_spec.rb

```diff
@@ -254,6 +254,9 @@ def load_response_fixture(path_suffix)
         create(:power_of_attorney_request, :with_acceptance, :with_form_submission,
                resolution_created_at: time_plus_one_day, poa_code:)
       end
+      let!(:replaced_request) do
```

Comment:
This looks like it's unused, did you mean to have it cover the `processed` logic change?

Reply:
This is used. Without the fix, the inclusion of this spec setup data results in the 500 we experienced in production.